src/appMain/routes/users/routes/UserDetail/components/KubeconfigValidity.js

```diff
@@ -97,13 +100,33 @@ const KubeconfigValidity = ({ settings, onSave, orgSetting, orgId }) => {
               margin="dense"
               id="seconds"
               name="seconds"
-              value={getHours(kubectlSettings?.validitySeconds)}
```

Comment:
why did we modify this from hours to minutes for kubeconfig validity ?

Reply:
This was changed because, we also wanted to allow the users to add time in decimals. So f user wants to set less than an hour or in-between hours that could be accommodated only if minutes.

Reply:
reverted